homeassistant/components/srp_energy/sensor.py

```diff
@@ -94,62 +109,61 @@ def __init__(self, coordinator):
         self._state = None
 
     @property
-    def name(self):
+    def name(self) -> str:
         """Return the name of the sensor."""
         return f"{DEFAULT_NAME} {self._name}"
 
     @property
-    def unique_id(self):
+    def unique_id(self) -> str:
         """Return sensor unique_id."""
         return self.type
 
     @property
-    def native_value(self):
+    def native_value(self) -> StateType:
         """Return the state of the device."""
         if self._state:
             return f"{self._state:.2f}"
         return None
 
     @property
-    def native_unit_of_measurement(self):
+    def native_unit_of_measurement(self) -> str:
         """Return the unit of measurement of this entity, if any."""
         return self._unit_of_measurement
 
     @property
-    def icon(self):
+    def icon(self) -> str:
         """Return icon."""
         return ICON
 
     @property
-    def usage(self):
+    def usage(self) -> str:
         """Return entity state."""
+        usage: float = 0.0
```

Comment:
We shouldn't default to 0. If we don't have any data we return `None` to represent unknown state.